orte/tools/orterun/orterun.1in

```diff
@@ -13,10 +13,12 @@
 .SH NAME
 .
 orterun, mpirun, mpiexec \- Execute serial and parallel jobs in Open MPI.
+oshrun, shmemrun \- Execute serial and parallel jobs in Open SHMEM.
```

Comment:
but orterun.1 has other synonyms (line 15) like mpirun,mpiexec
oshrun, shmemrun are not differ than mpirun.

The intent was to introduce mpirun synonyms.

do you think we should copy orterun.1 into oshrun.1 ?